ui/app/src/state/actions/dialogs.ts

```diff
@@ -188,6 +189,18 @@ export const closeUploadDialog = /*#__PURE__*/ createAction<StandardAction>('CLO
 export const uploadDialogClosed = /*#__PURE__*/ createAction('UPLOAD_DIALOG_CLOSED');
 // endregion
 
+// region Single File Upload Dialog
+export const showSingleFileUploadDialog = /*#__PURE__*/ createAction<Partial<SingleFileUploadDialogStateProps>>(
+  'SHOW_SINGLE_FILE_UPLOAD_DIALOG'
+);
+export const closeSingleFileUploadDialog = /*#__PURE__*/ createAction<StandardAction>(
+  'CLOSE_SINGLE_FILE_UPLOAD_DIALOG'
```

Comment:
There's an epic that listens from pretty much all `close.*Dialog`. You need to add this event to that epic's `ofType()`